packages/@lwc/aria-reflection-polyfill/src/detect.ts

```diff
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
+import { getOwnPropertyDescriptor } from '@lwc/shared';
+
 export function detect(propName: string): boolean {
-    return Object.getOwnPropertyDescriptor(Element.prototype, propName) === undefined;
+    return getOwnPropertyDescriptor(Element.prototype, propName) === undefined;
```

Comment:
Figured I might as well make this change. Makes the bundle size slightly smaller.